Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -858,11 +854,10 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Searches for additional alerts in the incident that may further
-        indicate user attempts to enumerate Active Directory.
+      description: Searches for additional alerts in the alert that may further indicate user attempts to enumerate Active Directory.
```

Comment:
Does **alerts in the alert** make sense?

Reply:
no, it was replaced automatically, fixed it